### PR TITLE
Add custom php settings to roundcubemail php-fpm pool

### PR DESCRIPTION
**Pull request by stephdl**
The PR attempts to adjust php settings to the php-fpm pool of rouncubemail and enable the roundcubemail mariadb server at boot 

```
roundcubemail=configuration
    MaxExecutionTime=0
    MaxFileUploads=20
    MemoryLimit=512
    PluginsList=managesieve,markasjunk
    PostMaxSize=32
    Server=localhost
    UploadMaxFilesize=32
    access=public
    skin=larry
```

https://github.com/NethServer/dev/issues/6666

### CODE REVIEW

File: root/etc/e-smith/templates/etc/opt/rh/rh-php73/php-fpm.d/000-roundcubemail.conf/20PhpSettings [hunk 1]
```diff
@@ -0,0 +1,19 @@
+; PHP settings
+
+{
+use esmith::ConfigDB;
+    my $db = esmith::ConfigDB->open_ro() || die("Can't open config db");
+
+            my $MemoryLimit = $db->get_prop('roundcubemail','MemoryLimit') || '512';
+            my $UpMaxFileSize = $db->get_prop('roundcubemail','UploadMaxFilesize') || '32';
```
Comment (gsanchietti) [inline root/etc/e-smith/templates/etc/opt/rh/rh-php73/php-fpm.d/000-roundcubemail.conf/20PhpSettings]: The indentation is a bit messy :)

File: root/etc/e-smith/db/configuration/defaults/roundcubemail/MaxExecutionTime [hunk 2]
```diff
@@ -0,0 +1 @@
+0
```
Comment (gsanchietti) [inline root/etc/e-smith/db/configuration/defaults/roundcubemail/MaxExecutionTime]: Maybe a value around 300 is better.

**Review by gsanchietti — COMMENTED** [review]
If you want, you can omit [root/etc/e-smith/db/configuration/defaults/rh-mariadb105-mariadb@roundcubemail/access](https://github.com/NethServer/nethserver-roundcubemail-next/pull/5/files#diff-ac11fdff2c8339640b93afcddbc8ce8be176408ab47fa72605950210c51c4077)

File: root/etc/e-smith/templates/etc/opt/rh/rh-php73/php-fpm.d/000-roundcubemail.conf/20PhpSettings (hunk 1 above)
Comment (stephdl) [reply root/etc/e-smith/templates/etc/opt/rh/rh-php73/php-fpm.d/000-roundcubemail.conf/20PhpSettings]: done

File: root/etc/e-smith/db/configuration/defaults/roundcubemail/MaxExecutionTime (hunk 2 above)
Comment (stephdl) [reply root/etc/e-smith/db/configuration/defaults/roundcubemail/MaxExecutionTime]: done

File: root/etc/e-smith/events/actions/nethserver-rouncubemail-next-clean-cache [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# clean the opcache of roundcubemail
+
+/usr/bin/rm `/usr/bin/find  /var/opt/rh/rh-php*/lib/php/opcache/ -name 'roundcubemail'` -rf
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-rouncubemail-next-clean-cache]: Please use find syntax:
```
find <path> -name <pattern> -delete
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]

File: root/etc/e-smith/events/actions/nethserver-rouncubemail-next-clean-cache (hunk 3 above)
Comment (stephdl) [reply root/etc/e-smith/events/actions/nethserver-rouncubemail-next-clean-cache]: I cannot remove not empty folder :'(

Comment (stephdl) [reply root/etc/e-smith/events/actions/nethserver-rouncubemail-next-clean-cache]: /usr/bin/find  /var/opt/rh/rh-php*/lib/php/opcache/ -name 'roundcubemail' -exec rm -rf {} +

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**